app/serializers/press-release.js

```diff
@@ -2,7 +2,7 @@ import ApplicationSerializer from './application';
 
 export default class PressReleaseSerializer extends ApplicationSerializer {
   shouldSerializeHasMany(snapshot, key/*, relationshipType*/) {
-    if (key == 'publicationChannels')
+    if (key == 'publicationChannels' || 'organization')
```

Comment:
Dit lijkt me vreemd omdat `organization` geen `hasMany` relatie is op persbericht. Het opslaan zou dus ook moeten werken zonder deze toevoeging.